### PR TITLE
5.3.1 prep

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add entry for coming release

# Testing this PR

Nothing to test

# Related reading

https://trello.com/b/2V7YCJKj/omero-5-3-1

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+
+Improvements include:
+
+- allow to view images in share
+- allow to import hidden image files (Windows client issue)
+- clarify installation of OMERO.web
+- save polygon and polyline as defined in the OME model
+
+This release also upgrades the version of Bio-Formats which OMERO uses to
+`5.4.1 <https://www.openmicroscopy.org/site/support/bio-formats5.4/about/whats-new.html>`_.
```
Comment (mtbc) [inline history.txt]: must merge https://github.com/openmicroscopy/openmicroscopy/pull/5251

Comment (jburel) [reply history.txt]: yes
I will add another for saving of "big image" as jpeg too i.e. https://github.com/openmicroscopy/openmicroscopy/pull/5212

**Comment by mtbc** [comment]
Looks  good so far at least. Happy to look again when final.

**Review by mtbc — APPROVED** [review]
#5256 is in

**Comment by jburel** [comment]
Thanks @mtbc

**Pull request merged**